v2/internal/crdtests/aks_managedcluster_20230102preview_test.go

```diff
@@ -3,7 +3,7 @@ Copyright (c) Microsoft Corporation.
 Licensed under the MIT license.
 */
 
-package controllers_test
+package crdtests
```

Comment:
minor: I _think_ we want this to be a `_test` package as the compiler treats those specially

Reply:
The only special behaviour that I'm aware of is that a `_test` package can coexist in the same folder as a normal one. This is useful to stop tests accessing internals, etc; not something we need here.

I'll fix the name.